### PR TITLE
Feat #2877: updated the docs of into-api.md

### LINKED ISSUE #2877: Update Postman API getting started experience
This page is out of date: https://learning.postman.com/docs/developer/intro-api/

The Postman API documentation now redirects to this public workspace: https://www.postman.com/postman/workspace/postman-public-workspace/documentation/12959542-c8142d51-e97c-46b6-bd77-52bb66712c9a

- Users get started by forking the collection (instead of the Run in Postman button import described on this page https://learning.postman.com/docs/developer/intro-api/)
- The web dashboard experience has changed, so getting an API key from the home page can be removed or updated

### PR DESCRIPTION
**Pull request by Cronus1007**
## Fixes
Fixes #2877 by @loopDelicious 

## Description 
Updated the documentation as per the instructions given in the issue

## Additional Context 
I would like to add two images for more details regarding the forking for more clarity to the end user but the images in the repo are controlled via `https://assets.postman.com/postman-docs` so gave these images in the description of the PR.
oie_DW0nvLWakotY

## Tests

NO need of testing .

### CODE REVIEW

File: src/pages/docs/developer/intro-api.md [hunk 1]
```diff
@@ -19,11 +19,11 @@ warning: false
 
 ---
 
-The [Postman API](https://docs.api.getpostman.com/) endpoints to help you integrate Postman within your development toolchain.
+The [Postman API](https://www.postman.com/postman/workspace/postman-public-workspace/documentation/12959542-c8142d51-e97c-46b6-bd77-52bb66712c9a) endpoints to help you integrate Postman within your development toolchain.
 
 You can add new collections, update existing collections, update environments, and add and run monitors directly through the API. This enables you to programmatically access data stored in your Postman account.
 
-You can get started with the API via the **Run in Postman** button at the top of the [Postman API documentation](https://docs.api.getpostman.com/) and use the Postman app to send requests.
+You can get started by forking the collection. For more details regarding the __Forking__ navigate to the [Postman API](https://www.postman.com/postman/workspace/postman-public-workspace/documentation/12959542-c8142d51-e97c-46b6-bd77-52bb66712c9a)
```
Comment (sowmyaraj92) [inline src/pages/docs/developer/intro-api.md:26]: A missing period at the end of the sentence.

File: src/pages/docs/developer/intro-api.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ warning: false
 
 ---
 
-The [Postman API](https://docs.api.getpostman.com/) endpoints to help you integrate Postman within your development toolchain.
+The [Postman API](https://www.postman.com/postman/workspace/postman-public-workspace/documentation/12959542-c8142d51-e97c-46b6-bd77-52bb66712c9a) endpoints to help you integrate Postman within your development toolchain.
```
Comment (sowmyaraj92) [inline src/pages/docs/developer/intro-api.md:22]: Change "endpoints to help" > "endpoints help you integrate".

Comment (sowmyaraj92) [inline src/pages/docs/developer/intro-api.md:22]: Change "endpoints to help" > "endpoints help you to integrate".

File: src/pages/docs/developer/intro-api.md (hunk 1 above)
Comment (sowmyaraj92) [inline src/pages/docs/developer/intro-api.md:26]: A missing period at the end of the sentence.

**Review by sowmyaraj92 — CHANGES_REQUESTED** [review]
Just a few minor changes and we are good to go!

**Review by sowmyaraj92 — APPROVED** [review]
@Cronus1007 Looks great! 👍

**Review by tristandenyer — APPROVED** [review]

**Comment by Cronus1007** [comment]
@sowmyaraj92 @tristandenyer Thanks.

**Pull request merged**